server/controllers/admin-post.js

```diff
@@ -34,4 +35,3 @@ exports.deletePost = (req, res) => {
     else res.status(200).json({ message: 'post deleted' });
   });
 };
-
```

Comment:
Don't remove this ending blank line. This is also suggested by eslint. I suggest you use an eslint plugin for your editor.